test/regressions/index.test.js

```diff
@@ -77,10 +90,33 @@ async function main() {
         await fse.ensureDir(path.dirname(screenshotPath));
         // Testcase.screenshot would resize the viewport to the element bbox.
         await page.screenshot({ clip, path: screenshotPath, type: 'png' });
+
+        if (errorConsole) {
+          const msg = await errorConsole;
+          errorConsole = undefined;
+          // eslint-disable-next-line no-console
+          console.log(`⚠️ Error logged in the console in ${pathURL}:\n`);
+          // eslint-disable-next-line no-console
+          console.log(msg.join('\n'));
+          errorConsoleCounter += 1;
+        }
```

Comment:
Why not expect no messages so that we immediately know where the error happened?

Reply:
With the `ERROR_THRESHOLD` tradeoff, we couldn't fail the test case directly. Different story now